lib/routes/thewirehindi/utils.ts

```diff
@@ -0,0 +1,36 @@
+import { DataItem } from '@/types';
+import { parseDate } from '@/utils/parse-date';
+import { art } from '@/utils/render';
+import path from 'node:path';
+import { getCurrentPath } from '@/utils/helpers';
+
+const __dirname = getCurrentPath(import.meta.url);
```

Comment:
No need to provide this variable as it will be provided by tsdown when transpiling https://github.com/DIYgod/RSSHub/blob/f455f542ab47715b935f982ccbe2ba1ce87bd343/tsdown.config.ts#L7